docs/source/getting-started/examples/histograms.ipynb

```diff
@@ -178,11 +186,11 @@
     "    d_in, d_out\n",
     ") -> dp.Measurement:\n",
     "    \"\"\"Make a stability histogram that respects a given d_in, d_out.\"\"\"\n",
-    "    def privatize(s, t=1e8):\n",
+    "    def privatize(s, t=100_000):\n",
     "        return preprocess >> dp.m.then_laplace_threshold(scale=s, threshold=t)\n",
     "    \n",
     "    s = dp.binary_search(lambda s: privatize(s=s).map(d_in)[0] <= d_out[0])\n",
-    "    t = dp.binary_search(lambda t: privatize(s=s, t=t).map(d_in)[1] <= d_out[1])\n",
+    "    t = dp.binary_search(lambda t: privatize(s=s, t=t).map(d_in)[1] <= d_out[1], T=int)\n",
```

Comment:
If it doesn't run without this change, what's the error message we get now? Is it enough for the user to know what to change. (Though it's a little sad if we need to start requiring more boilerplate from the user.)